src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -221,45 +221,13 @@ Note that electrode positions SHOULD NOT be added to this file but to `*_electro
 
 The columns of the channels description table stored in `*_channels.tsv` are:
 
-MUST be present **in this specific order**:
-
 <!-- This block generates a columns table.
 The definitions of these fields can be found in
-  src/schema/objects/columns.yaml
+  src/schema/rules/tabular_data/*.yaml
 and a guide for using macros can be found at
  https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
 -->
-{{ MACROS___make_columns_table(
-   {
-      "name__channels": ("REQUIRED", "When a corresponding electrode is specified in `_electrodes.tsv`, "
-                         "the name of that electrode MAY be specified here and the reference electrode "
-                         "name MAY be provided in the `reference` column."),
```

Comment:
This description addendum is missing from `rules.tabular_data.ieeg.iEEGChannels`.

Reply:
It looks like you handled this so I'll resolve it.